.travis.yml

```diff
@@ -2,46 +2,43 @@ sudo: false
 dist: trusty
 
 language: go
+go_import_path: gopkg.in/src-d/enry.v1
+
 go:
+  - '1.11.x'
   - '1.10.x'
-
-go_import_path: gopkg.in/src-d/enry.v1
+env:
+  global:
+    - GO_VERSION_FOR_JVM='1.11.x'
+  matrix:
+    - ONIGURUMA=0
+    - ONIGURUMA=1
+matrix:
+  fast_finish: true
 
 addons:
   apt:
     packages:
       - libonig-dev
 
-env:
-  global:
-    - GO_VERSION='1.10.x'
-    - ONIGURUMA=0
-
 stages:
   - name: test
   - name: release
     if: tag IS present
   - name: publish
     if: tag IS present
 
+stage: test
+install:
+  - if [[ -n "$ONIGURUMA" ]]; then tags="$tags oniguruma"; fi; go get -v -t --tags "$tags" ./...
+script:
+  - make test-coverage
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Should we vendor this script so that a network glitch doesn't cause our CI to stall?

I guess we already do this with Travis, but with gimme we're fetching from Travis's own mirrors. This seems like a third-party operation.

Reply:
I think it's their official API for integration https://docs.codecov.io/docs/supported-ci-providers#section-jenkins and all src-d projects use it this way.

Vendoring external resources is a good idea ofc, but I belive better be addressed in a separate issue, treating all resources the same way simultaneously.